example_site/src/components/Collections.js

```diff
@@ -96,7 +176,7 @@ function CollectionTemplates({
 
 export default function Collections() {
```

Comment:
I think we should rename this component and file now that it's redone to resemble our template browser - what do you think about `TemplateBrowser`? (very creative I know)